resources/js/components/Posts.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import Axios from "axios";
+import PostData from "./PostData";
+
+class Posts extends React.Component {
+    state = {
+        posts: [],
+        loading: true
+    };
+    fetchPosts = async () => {
+        const res = await Axios.get("/addPost");
+        if (res.data.status === 200) {
+            this.setState({ posts: res.data.posts });
+            this.setState({ loading: false });
+        }
```

Comment:
what if we don't have any posts? what if there was internal server error while loading posts?